modules/tinymce/src/core/test/ts/browser/keyboard/SpaceKeyTest.ts

```diff
@@ -109,5 +109,13 @@ describe('browser.tinymce.core.keyboard.SpaceKeyTest', () => {
       editor.selection.setContent(' ');
       TinyAssertions.assertContent(editor, '<p><span class="filler">s&nbsp;</span><span style="display: block;" contenteditable="false">a</span></p>');
     });
+
+    it('TINY-8588: Add one space before a block inside a strong', () => {
```

Comment:
Since the after is also change should there be a test for that as well as the before test. Seems that there are only tests for the before case on all of these should we have them on after the cef.

- [ ] Consider adding a test pressing space after the cef element